src/components/Auth/AuthHookForm.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+import { LinkbraryIcon } from "@/src/asset";
+import Link from "next/link";
+import { useForm } from "react-hook-form";
+import InputField from "./InputField/InputField";
+import styles from "./AuthHookForm.module.scss";
+
+export default function AuthHookForm() {
+  const { handleSubmit } = useForm();
+  // register : value 관련
+  // watch : input 상태감지
+  // formState : submit error 감지
+
+  return (
+    <main className={styles.authForm}>
+      <header className={styles.header}>
+        <Link href={"/"}>
+          <LinkbraryIcon width="212" height="38" />
+        </Link>
+        <div className={styles.header__text}>
+          <p>회원이 아니신가요?</p>
+          <Link href={"/signup"}>회원 가입하기</Link>
+        </div>
+      </header>
+      <section>
+        <form>
+          <InputField type="email" name="이메일" label={true} />
+          <InputField
+            type="password"
+            name="비밀번호"
+            icon="eyeToggle"
+            label={true}
```

Comment:
요기도 이렇게 해주면 좋을 것 같아요~
```suggestion
            label
```

Reply:
앗 확인해보니까 InputField 컴포넌트에서는 지금 label값은 사용하고 있지는 않네요!